src/main/resources/assets/api-docs.yaml

```diff
@@ -2819,7 +2819,7 @@ paths:
                 $ref: '#/components/schemas/ElectionReview'
   /api/electionReview/rp/{electionId}:
     get:
-      summary: describeElectionById
+      summary: getRPElectionReviewByReferenceId
```

Comment:
This should be a plain text shortened version of the description, not the method name. The method-name pattern was used by Belatrix, but we should update these to be more descriptive so the swagger UI is easier to read. Don't hold up the PR on this, so feel free to merge if you do update these text summaries.